### DISCUSSION TITLE
[Question]: How do I write clips matching detected events to a folder?

### DISCUSSION THREAD

**Discussion by reubenfirmin** [blakeblackshear/frigate#12445] [Ask A Question]
### What is your question:

frigate/storage/recordings has a firehose of recordings - I'm guessing this is the continual processing of the pipeline.

frigate/storage/clips has nothing, despite there being several handfuls of detected events, which are visible and downloadable from the UI.

How do I set frigate up so that clips are written when events are detected? I must be missing it in the docs. (I don't want timelapse - seems like that's what export wants to do).

I plan to run `inotifywait` on the directory and copy new files up to backblaze. I only want to do this with genuine events.

**Comment by NickM-27** [comment]
Exports can be done to accomplish this, there is no built in way to do this for frigate

**Reply by reubenfirmin** [reply]
> Exports can be done to accomplish this
 
What does this mean? The only export examples I see are with timelapse, and I don't want timelapse. Does just adding a barebone exports block start writing clips?

>  there is no built in way to do this for frigate

Is this on the roadmap? Offsite backup of events is a reasonably common need I would think. As long as they are dumped into a folder that frigate no longer cares about it should be easy for users to wire them up from there to whatever cloud storage is needed.

Thanks

**Reply by NickM-27** [reply]
> What does this mean? The only export examples I see are with timelapse, and I don't want timelapse. Does just adding a barebone exports block start writing clips?

there are multiple options being realtime and timelapse

> Offsite backup of events is a reasonably common need I would think. As long as they are dumped into a folder that frigate no longer cares about it should be easy for users to wire them up from there to whatever cloud storage is needed.

that can be done with exports it just requires an external script to handle calling the api
